### PR TITLE
[LPTOCPCI-247] Add the firewatch image and the firewatch-report-issues ref for Jira reporting

### PR DESCRIPTION
**Pull request by calebevans**
This PR adds some necessary items to the release repository for the usage of the [firewatch tool](https://github.com/CSPI-QE/firewatch):

1. A config to build and promote an image of the tool to be used in a ref
2. A ref that is to be used in a workflow or in tests to report issues to Jira based on failures in an OpenShift CI job
3. A workflow to replace the ipi-aws workflow, adding the firewatch ref mentioned in the above item to the end of the post steps.

### CODE REVIEW

File: clusters/app.ci/registry-access/cspi-qe/admin_oadp-qe.yaml [hunk 1]
```diff
@@ -0,0 +1,24 @@
+---
```
Comment (Simranpal) [inline clusters/app.ci/registry-access/cspi-qe/admin_oadp-qe.yaml:1]: I didnt understand the use of this file, and why reference to oadp qe in file name

File: ci-operator/step-registry/firewatch/report-issues/firewatch-report-issues-ref.yaml [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+    default: https://issues.stage.redhat.com
+    documentation: The Jira server issues are to be reported to.
+  - name: FIREWATCH_JIRA_API_TOKEN_PATH
+    default: /tmp/secrets/jira/access_token
+    documentation: The path to the file containing the Jira API token.
+  - name: FIREWATCH_FAIL_WITH_TEST_FAILURES
+    default: "false"
+    documentation: If you would like this ref to fail with a non-zero exit code if a test failure is found, set to "true"
+  - name: FIREWATCH_CONFIG
+    documentation: The firewatch rules to be used. See https://github.com/CSPI-QE/firewatch for more information on how to define this.
```
Comment (Simranpal) [inline ci-operator/step-registry/firewatch/report-issues/firewatch-report-issues-ref.yaml:27]: Shouldn't this have the default `/tmp/jira.config`

**Review by Simranpal — COMMENTED** [review]
Looks good. just some doubts I had

File: clusters/app.ci/registry-access/cspi-qe/admin_oadp-qe.yaml (hunk 1 above)
Comment (calebevans) [reply clusters/app.ci/registry-access/cspi-qe/admin_oadp-qe.yaml]: Ah, thank you for catching this! This file is used to define permissions in the ci registry for the cspi-qe namespace. I meant to rename this file as I used that OADP version of the file as a template.

File: ci-operator/step-registry/firewatch/report-issues/firewatch-report-issues-ref.yaml (hunk 2 above)
Comment (Simranpal) [reply ci-operator/step-registry/firewatch/report-issues/firewatch-report-issues-ref.yaml:27]: Okay I see, thanks!

**Review by ascerra — APPROVED** [review]

**Review by hongkailiu — COMMENTED** [review]
/approve

**Pull request merged**